docs/src/developing.md

```diff
@@ -3,28 +3,41 @@
 This section is designed to guide developers who wish to write their own graph structures.
 
 All LightGraphs functions rely on a standard API to function. As long as your graph structure is a subtype of
-[`AbstractGraph`](@ref) and implements the following API functions with the given return values, all functions
+[`AbstractGraph`](@ref) and implements the following API functions, all functions
 within the LightGraphs package should just work:
 
 - [`edges`](@ref)
 - [Base.eltype](https://docs.julialang.org/en/latest/base/collections/#Base.eltype)
 - [`edgetype`](@ref) (example: `edgetype(g::CustomGraph) = LightGraphs.SimpleEdge{eltype(g)})`)
 - [`has_edge`](@ref)
-- [`has_vertex`](@ref)
 - [`inneighbors`](@ref)
 - [`ne`](@ref)
 - [`nv`](@ref)
 - [`outneighbors`](@ref)
-- [`vertices`](@ref)
+- [`zero`](@ref)
 - [`is_directed`](@ref): Note that since LightGraphs uses traits to determine directedness, `is_directed` for a `CustomGraph` type
-should be implemented with **both** of the following signatures:
+must be implemented with the following signature:
   - `is_directed(::Type{CustomGraph})::Bool` (example: `is_directed(::Type{<:CustomGraph}) = false`)
+The following signature is optional:
   - `is_directed(g::CustomGraph)::Bool`
```

Comment:
Not strictly true: https://github.com/JuliaGraphs/LightGraphs.jl/blob/eb2eebeec09af5bfd63e46422f6941a4fe974757/src/Parallel/centrality/betweenness.jl#L17 and others

Reply:
oh, great. ok.